daemon/com.c

```diff
@@ -619,7 +619,7 @@ com_log_all_objects(
             {
                 if (object->thread_id_record_table[tid_index].lock_state != LOCK_STATE_NONE)
                 {
-                    log_message(MTC_LOG_DEBUG, "COM:     lock_state=%d thread_id=%x changed_time=%d(ms) .\n",
+                    log_message(MTC_LOG_DEBUG, "COM:     lock_state=%d thread_id=%lx changed_time=%lld(ms) .\n",
```

Comment:
I know this was preexstingly broken, but `0x%lx` as you're changing it.  It's never ok to mix hex and dec without identifying prefixes.

Reply:
`PRId64` from `inttypes.h`